cmd/kyma/alpha/create/module/module.go

```diff
@@ -24,9 +26,9 @@ func NewCmd(o *Options) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "module MODULE_NAME MODULE_VERSION <CONTENT_PATH> [flags]",
-		Short: "Creates a module bundled as an OCI image from the contents of the given path",
-		Long: `Use this command to create a Kyma module.
+		Use:   "module OCI_IMAGE_NAME MODULE_VERSION <CONTENT_PATH> [flags]",
+		Short: "Creates a module bundled as an OCI image with the given from the contents of the given path",
```

Comment:
"with the given from the contents" sounds weird

Reply:
Yes, I think I forgot a word in there...

Reply:
Fixed on previous PR to avoid more confusion on early adopters.